vault/resource_token.go

```diff
@@ -407,3 +419,11 @@ func tokenCheckLease(d *schema.ResourceData) bool {
 
 	return false
 }
+
+func metaFromMap(input map[string]interface{}) map[string]string {
+	result := make(map[string]string, len(input))
+	for k, v := range input {
+		result[k] = v.(string)
```

Comment:
If a user erroneously provides a bool or int here, the string that comes out of `v.(string)` will be `""` and the code will continue along. Would it be possible to instead fail here with explicit output saying that they must provide a string but they provided something else for X key?

Reply:
Ah yes! Thanks for the test coverage on that!